runtime/etc/Makefile.mli-static

```diff
@@ -127,6 +144,18 @@ ifneq ($(SKIP_COMPILE_LINK),skip)
 		-lchpl -lm \
 		-L$(CHPL_ZMQ_LOC)/lib \
 		-lzmq
+else
```

Comment:
To be honest, I'm not sure why it never occurred to me that we could just have the compiler search on the system library path. I think that it didn't work locally for me, which is why I added the variable.